src/framework/preferences/html/PrefsEditorTemplate-speak.html

```diff
@@ -1,11 +1,10 @@
 <h2><span class="fl-icon-speak"></span><span class="flc-prefsEditor-speak-label heading-text"></span></h2>
-
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
+    	<p class="flc-prefsEditor-speak-choice-label"></p>
```

Comment:
Wanted to have the label and toggle button on two different lines, changed it to a span class but added two <br> after. not sure if that's best practice given that it's an inline label element...? Perhaps this will be addressed in the other jira for moving to fieldset